tools/sro2kmm/sro2kmm

```diff
@@ -1,6 +1,5 @@
 #!/usr/bin/env bash
 #
-set -ex
```

Comment:
Don't we want to keep `-e`? How about `-u`?

Reply:
Done.